Buildings/Resources/Scripts/Dymola/Experimental/DHC/CentralPlants/Cooling/Subsystems/Examples/CoolingTowerParallel.mos

```diff
@@ -0,0 +1,39 @@
+﻿// Simulate
```

Comment:
The unit tests produced slightly different results than the reference results for this model. Could you please verify the correct results and update the .txt file as needed?
![image](https://user-images.githubusercontent.com/44682485/99437677-0a892200-28d0-11eb-91a4-67d9d525eb0e.png)

Reply:
Updated